### PR TITLE
Connectivity sleep schema

### LINKED ISSUE #9: Schemat łączności
Diagram systemowy w jaki sposób działa łączenie się serwera z klientem.

Chodziło o to:

- po włączeniu klienta - rpi konfiguruje synchronizację do budzenia się esp
- co jakiś czas esp budzi się z light sleep (research needed! - sleep mode który pozwala na budzenie zegarem) i łączy się z serwerem

### PR DESCRIPTION
**Pull request by wojtekszamocki**

### CODE REVIEW

**Comment by olaaaf** [comment]
Git, ale waked up to woke up

**Comment by D-Mielewczyk** [comment]
@olaaaf to nie akcpetuj jak jest błąd!!

File: docs/assets/uml/serwer_klient.png
Comment (D-Mielewczyk) [inline docs/assets/uml/serwer_klient.png:1]: speeling waked_up -> woke_up

File: graphor/serwer_klient_sleep.gaphor
Comment (D-Mielewczyk) [inline graphor/serwer_klient_sleep.gaphor:1]: same spelling error

**Review by D-Mielewczyk — CHANGES_REQUESTED** [review]

File: graphor/serwer_klient_sleep.gaphor
Comment (wojtekszamocki) [reply graphor/serwer_klient_sleep.gaphor:1]: Done in  https://github.com/groupplants/docs/pull/25/commits/660a8fdd3fd6f9296eea81ecf153962a1465fbac

**Review by D-Mielewczyk — APPROVED** [review]

**Pull request merged**